helloWorld/settings.py

```diff
@@ -46,6 +46,7 @@
 ]
 
 MIDDLEWARE_CLASSES = [
+    'htmlmin.middleware.HtmlMinifyMiddleware',
```

Comment:
As this is a React-mounted application and we aren't intending to implement server-side rendering, the HTML content we have atm is final. Hence, there isn't any need to scale.

But anyway, there is no impact of this plugin, so I'll merge this!